_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -75545,7 +75545,7 @@
 /turf/simulated/floor/plasteel,
 /area/station/service/janitor)
 "mSV" = (
-/mob/living/simple_animal/hostile/feral_cat/forsaken,
```

Comment:
/mob/living/simple_animal/hostile/feral_cat/forsaken isnt used anywhere else in the game, can you remove it please?

Reply:
Removed the entry